[2022-06-11 18:57:59,560] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: T-dbt-job-objects.dbt_clone_and_transform-objects manual__2022-06-11T18:57:59.266373+00:00 [queued]>
[2022-06-11 18:57:59,577] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: T-dbt-job-objects.dbt_clone_and_transform-objects manual__2022-06-11T18:57:59.266373+00:00 [queued]>
[2022-06-11 18:57:59,578] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-06-11 18:57:59,578] {taskinstance.py:1357} INFO - Starting attempt 1 of 2
[2022-06-11 18:57:59,578] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-06-11 18:57:59,597] {taskinstance.py:1377} INFO - Executing <Task(BashOperator): dbt_clone_and_transform-objects> on 2022-06-11 18:57:59.266373+00:00
[2022-06-11 18:57:59,606] {standard_task_runner.py:52} INFO - Started process 1877 to run task
[2022-06-11 18:57:59,612] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'T-dbt-job-objects', 'dbt_clone_and_transform-objects', 'manual__2022-06-11T18:57:59.266373+00:00', '--job-id', '113', '--raw', '--subdir', 'DAGS_FOLDER/docker_job/dbt_model.py', '--cfg-path', '/tmp/tmpyv01kh3g', '--error-file', '/tmp/tmp15nm7rvb']
[2022-06-11 18:57:59,613] {standard_task_runner.py:80} INFO - Job 113: Subtask dbt_clone_and_transform-objects
[2022-06-11 18:57:59,715] {task_command.py:370} INFO - Running <TaskInstance: T-dbt-job-objects.dbt_clone_and_transform-objects manual__2022-06-11T18:57:59.266373+00:00 [running]> on host d8836eb746ac
[2022-06-11 18:57:59,802] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=samuelolle@yahoo.com
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=T-dbt-job-objects
AIRFLOW_CTX_TASK_ID=dbt_clone_and_transform-objects
AIRFLOW_CTX_EXECUTION_DATE=2022-06-11T18:57:59.266373+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-06-11T18:57:59.266373+00:00
[2022-06-11 18:57:59,804] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-06-11 18:57:59,804] {subprocess.py:74} INFO - Running command: ['bash', '-c', " rm -rf /tmp/objects && mkdir /tmp/objects                            && cp -r /tmp/dbt /tmp/objects/dbt                             && cd /tmp/objects/dbt && python3 -m venv venv                             && export PIP_USER=false                             && *** /tmp/objects/dbt/venv/bin/activate                             && pip  install --quiet --upgrade pip setuptools                             && pip3 install --quiet 'MarkupSafe<=2.0.1' 'dbt-postgres' 'dbt-core'                             && dbt --version && deactivate                             && cd /tmp/objects/dbt && *** /tmp/objects/dbt/venv/bin/activate                             && dbt deps --profiles-dir /tmp/objects/dbt/ && dbt seed --profiles-dir /tmp/objects/dbt/                             && dbt run --models objects --profiles-dir /tmp/objects/dbt/"]
[2022-06-11 18:57:59,826] {subprocess.py:85} INFO - Output:
[2022-06-11 18:58:37,012] {subprocess.py:92} INFO - Core:
[2022-06-11 18:58:37,012] {subprocess.py:92} INFO -   - installed: 1.1.0
[2022-06-11 18:58:37,012] {subprocess.py:92} INFO -   - latest:    1.1.0 - [32mUp to date![0m
[2022-06-11 18:58:37,012] {subprocess.py:92} INFO - 
[2022-06-11 18:58:37,012] {subprocess.py:92} INFO - Plugins:
[2022-06-11 18:58:37,012] {subprocess.py:92} INFO -   - postgres: 1.1.0 - [32mUp to date![0m
[2022-06-11 18:58:39,805] {subprocess.py:92} INFO - [0m18:58:39  Running with dbt=1.1.0
[2022-06-11 18:58:39,807] {subprocess.py:92} INFO - [0m18:58:39  Warning: No packages were found in packages.yml
[2022-06-11 18:58:42,690] {subprocess.py:92} INFO - [0m18:58:42  Running with dbt=1.1.0
[2022-06-11 18:58:42,706] {subprocess.py:92} INFO - [0m18:58:42  Partial parse save file not found. Starting full parse.
[2022-06-11 18:58:43,312] {subprocess.py:92} INFO - [0m18:58:43  [WARNING]: Configuration paths exist in your dbt_project.yml file which do not apply to any re***s.
[2022-06-11 18:58:43,313] {subprocess.py:92} INFO - There are 1 unused configuration paths:
[2022-06-11 18:58:43,313] {subprocess.py:92} INFO - - seeds.SAM
[2022-06-11 18:58:43,313] {subprocess.py:92} INFO - 
[2022-06-11 18:58:43,324] {subprocess.py:92} INFO - [0m18:58:43  Found 4 models, 6 tests, 0 snapshots, 0 analyses, 171 macros, 4 operations, 0 seed files, 4 ***s, 0 exposures, 0 metrics
[2022-06-11 18:58:43,325] {subprocess.py:92} INFO - [0m18:58:43
[2022-06-11 18:58:43,326] {subprocess.py:92} INFO - [0m18:58:43  [WARNING]: Nothing to do. Try checking your model configs and model specification args
[2022-06-11 18:58:43,330] {subprocess.py:92} INFO - [0m18:58:43
[2022-06-11 18:58:43,331] {subprocess.py:92} INFO - [0m18:58:43  Completed successfully
[2022-06-11 18:58:43,331] {subprocess.py:92} INFO - [0m18:58:43
[2022-06-11 18:58:43,331] {subprocess.py:92} INFO - [0m18:58:43  Done. PASS=0 WARN=0 ERROR=0 SKIP=0 TOTAL=0
[2022-06-11 18:58:46,210] {subprocess.py:92} INFO - [0m18:58:46  Running with dbt=1.1.0
[2022-06-11 18:58:46,247] {subprocess.py:92} INFO - [0m18:58:46  [WARNING]: Configuration paths exist in your dbt_project.yml file which do not apply to any re***s.
[2022-06-11 18:58:46,247] {subprocess.py:92} INFO - There are 1 unused configuration paths:
[2022-06-11 18:58:46,247] {subprocess.py:92} INFO - - seeds.SAM
[2022-06-11 18:58:46,247] {subprocess.py:92} INFO - 
[2022-06-11 18:58:46,263] {subprocess.py:92} INFO - [0m18:58:46  Found 4 models, 6 tests, 0 snapshots, 0 analyses, 171 macros, 4 operations, 0 seed files, 4 ***s, 0 exposures, 0 metrics
[2022-06-11 18:58:46,265] {subprocess.py:92} INFO - [0m18:58:46
[2022-06-11 18:58:46,333] {subprocess.py:92} INFO - [0m18:58:46  Concurrency: 4 threads (target='dev')
[2022-06-11 18:58:46,334] {subprocess.py:92} INFO - [0m18:58:46
[2022-06-11 18:58:46,343] {subprocess.py:92} INFO - [0m18:58:46  1 of 1 START table model analytics_object.obj_walls ............................ [RUN]
[2022-06-11 18:58:46,449] {subprocess.py:92} INFO - [0m18:58:46  1 of 1 OK created table model analytics_object.obj_walls ....................... [SELECT 36 in 0.10s]
[2022-06-11 18:58:46,551] {subprocess.py:92} INFO - [0m18:58:46
[2022-06-11 18:58:46,552] {subprocess.py:92} INFO - [0m18:58:46  Running 4 on-run-end hooks
[2022-06-11 18:58:46,558] {subprocess.py:92} INFO - [0m18:58:46  1 of 4 START hook: SAM.on-run-end.0 ............................................ [RUN]
[2022-06-11 18:58:46,561] {subprocess.py:92} INFO - [0m18:58:46  1 of 4 OK hook: SAM.on-run-end.0 ............................................... [ALTER DEFAULT PRIVILEGES in 0.00s]
[2022-06-11 18:58:46,565] {subprocess.py:92} INFO - [0m18:58:46  2 of 4 START hook: SAM.on-run-end.1 ............................................ [RUN]
[2022-06-11 18:58:46,567] {subprocess.py:92} INFO - [0m18:58:46  2 of 4 OK hook: SAM.on-run-end.1 ............................................... [ALTER DEFAULT PRIVILEGES in 0.00s]
[2022-06-11 18:58:46,570] {subprocess.py:92} INFO - [0m18:58:46  3 of 4 START hook: SAM.on-run-end.2 ............................................ [RUN]
[2022-06-11 18:58:46,572] {subprocess.py:92} INFO - [0m18:58:46  3 of 4 OK hook: SAM.on-run-end.2 ............................................... [ALTER DEFAULT PRIVILEGES in 0.00s]
[2022-06-11 18:58:46,576] {subprocess.py:92} INFO - [0m18:58:46  4 of 4 START hook: SAM.on-run-end.3 ............................................ [RUN]
[2022-06-11 18:58:46,578] {subprocess.py:92} INFO - [0m18:58:46  4 of 4 OK hook: SAM.on-run-end.3 ............................................... [ALTER DEFAULT PRIVILEGES in 0.00s]
[2022-06-11 18:58:46,579] {subprocess.py:92} INFO - [0m18:58:46
[2022-06-11 18:58:46,580] {subprocess.py:92} INFO - [0m18:58:46
[2022-06-11 18:58:46,580] {subprocess.py:92} INFO - [0m18:58:46  Finished running 1 table model, 4 hooks in 0.31s.
[2022-06-11 18:58:46,587] {subprocess.py:92} INFO - [0m18:58:46
[2022-06-11 18:58:46,587] {subprocess.py:92} INFO - [0m18:58:46  Completed successfully
[2022-06-11 18:58:46,587] {subprocess.py:92} INFO - [0m18:58:46
[2022-06-11 18:58:46,588] {subprocess.py:92} INFO - [0m18:58:46  Done. PASS=1 WARN=0 ERROR=0 SKIP=0 TOTAL=1
[2022-06-11 18:58:46,761] {subprocess.py:96} INFO - Command exited with return code 0
[2022-06-11 18:58:46,791] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=T-dbt-job-objects, task_id=dbt_clone_and_transform-objects, execution_date=20220611T185759, start_date=20220611T185759, end_date=20220611T185846
[2022-06-11 18:58:46,818] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-06-11 18:58:46,845] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
